vertx-mysql-client/src/main/java/io/vertx/mysqlclient/MySQLChangeUserOptions.java

```diff
@@ -0,0 +1,249 @@
+    this.password = other.password;
+    this.database = other.database;
+    this.collation = other.collation;
+    this.charset = other.charset;
+    this.serverRsaPublicKeyPath = other.serverRsaPublicKeyPath;
+    this.serverRsaPublicKeyValue = other.serverRsaPublicKeyValue != null ? other.serverRsaPublicKeyValue.copy() : null;
+    this.properties = new HashMap<>(other.properties);
+  }
+
+  /**
+   * Get the user account to be used for the authentication.
+   *
+   * @return the user
+   */
+  public String getUser() {
+    return user;
+  }
+
+  /**
+   * Specify the user account to be used for the authentication.
+   *
+   * @param user the user to specify
+   * @return a reference to this, so the API can be used fluently
+   */
+  public MySQLChangeUserOptions setUser(String user) {
+    checkParameterNonNull(user, "User account can not be null");
+    this.user = user;
+    return this;
+  }
+
+  /**
+   * Get the user password to be used for the authentication.
+   *
+   * @return the password
+   */
+  public String getPassword() {
+    return password;
+  }
+
+  /**
+   * Specify the user password to be used for the authentication.
+   *
+   * @param password the password to specify
+   * @return a reference to this, so the API can be used fluently
+   */
+  public MySQLChangeUserOptions setPassword(String password) {
+    checkParameterNonNull(password, "Password can not be null");
+    this.password = password;
+    return this;
+  }
+
+  /**
+   * Get the database name for the re-authentication.
+   *
+   * @return the database name
+   */
+  public String getDatabase() {
+    return database;
+  }
+
+  /**
+   * Specify the default database for the re-authentication.
+   *
+   * @param database the database name to specify
+   * @return a reference to this, so the API can be used fluently
+   */
+  public MySQLChangeUserOptions setDatabase(String database) {
+    checkParameterNonNull(database, "Database name can not be null");
+    this.database = database;
+    return this;
+  }
+
+  /**
+   * @return the value of current connection attributes
+   */
+  public Map<String, String> getProperties() {
+    return properties;
+  }
+
+  /**
+   * Set connection attributes which will be sent to server at the re-authentication.
+   *
+   * @param properties the value of properties to specify
+   * @return a reference to this, so the API can be used fluently
+   */
+  public MySQLChangeUserOptions setProperties(Map<String, String> properties) {
+    checkParameterNonNull(properties, "Properties can not be null");
+    this.properties = properties;
+    return this;
+  }
+
+  /**
+   * Add a property for this client, which will be sent to server at the re-authentication.
+   *
+   * @param key   the value of property key
+   * @param value the value of property value
+   * @return a reference to this, so the API can be used fluently
+   */
+  public MySQLChangeUserOptions addProperty(String key, String value) {
```

Comment:
I think we need to annotate this with `@GenIgnore` because it generates a duplicate converter called `propertys`

Reply:
added in https://github.com/eclipse-vertx/vertx-sql-client/pull/420/commits/4145bd592180460c67cdbbb7e8efff435ad3d830.